src/tcgwars/logic/impl/gen3/HolonPhantoms.groovy

```diff
@@ -752,10 +773,12 @@ public enum HolonPhantoms implements LogicCardInfo {
           actionA {
             checkNoSPC()
             checkLastTurn()
+            //TODO: Check for it being single-counter poison, multiple should be overridable.
```

Comment:
The `GET_EXTRA_POISON` getter should help with that. Seems to be used in a couple of abilities already for reference.

Reply:
Tested and already commited, marking this as resolved.